pytest/integration/test_hbar_integration.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Integration tests to make sure hbar values set correctly in returned results"""
+import pytest
+
+import numpy as np
+
+import strawberryfields as sf
+from strawberryfields import ops
+
+
+N_MEAS = 200
+NUM_STDS = 10.0
+STD_10 = NUM_STDS / np.sqrt(N_MEAS)
+
+R = 0.3
+X = 0.2
+P = 0.1
+HBAR = [0.5, 1]
+
+
+@pytest.mark.parametrize("hbar", HBAR, indirect=True)
```

Comment:
:thinking: What does `indirect` do here?

Reply:
It's pretty cool, it allows you to parametrize an already defined fixture (in this case, `hbar`, which was set to 2 in conftest.py)